utils/encoding_utils.go

```diff
@@ -14,3 +15,28 @@ func Base64UrlDecode(data string) []byte {
 	decodedDataBytes, _ := rawURLEncoding.DecodeString(data)
 	return decodedDataBytes
 }
+
+func HexUrlEncode(data []byte) string {
+	return hex.EncodeToString(data)
+}
+
+func HexUrlDecode(data string) []byte {
+	decodedDataBytes, _ := hex.DecodeString(data)
+	return decodedDataBytes
+}
+
+func EncodeData(data []byte, encodingType string) string {
+	if encodingType == "BASE64" {
+		return Base64UrlEncode(data)
+	} else {
+		return HexUrlEncode(data)
+	}
+}
+
+func DecodeData(data string, encodingType string) []byte {
+	if encodingType == "BASE64" {
+		return Base64UrlDecode(data)
+	} else {
```

Comment:
can you please make the encoding type selection specific, and add error handing if non supported type is selected

Reply:
Added a check to 'field_level_encryption_config_builder' so we know at this stage that the encoding type will be either BASE64 or HEX